<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xSrstitch</class>
 <widget class="QDialog" name="xSrstitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>527</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>45</red>
        <green>45</green>
        <blue>45</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>45</red>
        <green>45</green>
        <blue>45</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>45</red>
        <green>45</green>
        <blue>45</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>36</red>
        <green>36</green>
        <blue>36</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Hack</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>xSrstitch</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/imgs/sew1.png</normalon>
   </iconset>
  </property>
  <widget class="QTextBrowser" name="pTextBrowser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>551</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>831</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="pL_InputDir">
      <property name="text">
       <string>Select Input Dir of Dirs</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="pLE_InputDir"/>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="pButton_InputDir">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="pL_OutputDir">
      <property name="text">
       <string>Select Output Dir</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="pLE_OutputDir"/>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="pButton_OutputDir">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>8</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>40</y>
      <width>261</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2" colspan="3">
      <widget class="QDoubleSpinBox" name="pSpinBox_voxelsize">
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="pL_Match">
       <property name="text">
        <string>matching from start</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3" colspan="2">
      <widget class="QSpinBox" name="pSpinBox_Match"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="pL_Range">
       <property name="text">
        <string>set search Range</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="pL_voxelsize">
       <property name="text">
        <string>Voxel size [µm]</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pL_stackOverlap">
       <property name="text">
        <string>Substack feed [mm]</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3" colspan="2">
      <widget class="QSpinBox" name="pSpinBox_Range"/>
     </item>
     <item row="1" column="2" colspan="3">
      <widget class="QDoubleSpinBox" name="pSpinBox_stackOverlap">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" rowspan="3" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" rowspan="3">
      <widget class="QLabel" name="pL_Ref">
       <property name="text">
        <string>reference from bottom</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" rowspan="3" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="3" rowspan="3" colspan="2">
      <widget class="QSpinBox" name="pSpinBox_Ref"/>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="pRadioLock">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>218</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="data.qrc">
      <normaloff>:/imgs/lock_closed.png</normaloff>
      <normalon>:/imgs/lock_open.png</normalon>:/imgs/lock_closed.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>271</width>
     <height>78</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pButton_slicePreview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>44</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>show Preview slices</string>
    </property>
   </widget>
   <widget class="QLabel" name="pL_selectSubFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>select Preview Subfolder</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pComboBox_selectSubFolder">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>15</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>831</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_Close">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>470</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>8</pointsize>
     <italic>true</italic>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>461</width>
     <height>56</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>189</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pButton_Plot">
      <property name="text">
       <string>plot corr</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pButton_Go">
      <property name="font">
       <font>
        <family>Hack</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>calculate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pButtonShowImgs">
      <property name="text">
       <string>showIMGs</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pButton_Stitch">
      <property name="font">
       <font>
        <family>Hack</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>stitch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pLE_InputDir</tabstop>
  <tabstop>pButton_InputDir</tabstop>
  <tabstop>pLE_OutputDir</tabstop>
  <tabstop>pButton_OutputDir</tabstop>
  <tabstop>pSpinBox_voxelsize</tabstop>
  <tabstop>pSpinBox_stackOverlap</tabstop>
  <tabstop>pSpinBox_Ref</tabstop>
  <tabstop>pSpinBox_Match</tabstop>
  <tabstop>pSpinBox_Range</tabstop>
  <tabstop>pRadioLock</tabstop>
  <tabstop>pComboBox_selectSubFolder</tabstop>
  <tabstop>pButton_slicePreview</tabstop>
  <tabstop>pButton_Close</tabstop>
  <tabstop>pTextBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections/>
</ui>
